mlrun/runtimes/mpijob/v1alpha1.py

```diff
@@ -73,8 +73,8 @@ def _generate_mpi_job(self, runobj: RunObject, execution: MLClientCtx, meta: cli
         self._update_container(job, 'volumeMounts', self.spec.volume_mounts)
 
         extra_env = {'MLRUN_EXEC_CONFIG': runobj.to_json()}
-        # if self.spec.rundb:
-        #     extra_env['MLRUN_DBPATH'] = self.spec.rundb
+        if runobj.spec.verbose:
+            extra_env['MLRUN_LOG_LEVEL'] = 'debug'
```

Comment:
instead of repeating this everywhere I suggest to move it to a function in the base class